### PR TITLE
FIX: Remove attrs that do not belong to plugins.

### PR DESCRIPTION
**Pull request by danielballan**
This is after length discussion with and assistance from @dchabot and @arkilic. We are pretty sure that these fields do not *belong* to the plugins; they are just used by them. They should not be exposed per-plugin.

Also see:

```python
In [42]: for attr in dir(det.tiff1):
    try:
            getattr(det.tiff1, attr)
                except ophyd.utils.TimeoutError:
                        print(attr)
                           ....:
pool_alloc_buffers
pool_free_buffers
pool_max_buffers
pool_max_mem
pool_used_buffers
pool_used_mem
queue_free
queue_free_low
queue_size
queue_use
queue_use_high
queue_use_hihi
write_message
write_status
```

Those last two actually *do* belong to the fileplugin -- they are just failing because of an issue with the test IOC.

### CODE REVIEW

**Comment by arkilic** [comment]
Come back and merge this @klauer

File: ophyd/controls/areadetector/plugins.py [hunk 1]
```diff
@@ -66,13 +66,6 @@ class PluginBase(ADBase):
     asyn_io = C(EpicsSignal, 'AsynIO')
 
     nd_attributes_file = C(EpicsSignal, 'NDAttributesFile', string=True)
-    pool_alloc_buffers = C(EpicsSignalRO, 'PoolAllocBuffers')
-    pool_free_buffers = C(EpicsSignalRO, 'PoolFreeBuffers')
-    pool_max_buffers = C(EpicsSignalRO, 'PoolMaxBuffers')
-    pool_max_mem = C(EpicsSignalRO, 'PoolMaxMem')
-    pool_used_buffers = C(EpicsSignalRO, 'PoolUsedBuffers')
-    pool_used_mem = C(EpicsSignalRO, 'PoolUsedMem')
-    port_name = C(EpicsSignalRO, 'PortName_RBV', string=True)
```
Comment (klauer) [inline ophyd/controls/areadetector/plugins.py]: `port_name` does not belong in this PR, it is very useful

**Comment by klauer** [comment]
I'm OK with removing these from the plugins, but I'd like to still have them accessible in the future for debugging purposes.

Perhaps create a `ophyd.controls.areadetector.debug` module and have a set of `Pool`/`Queue` Devices or something along those lines.

**Comment by danielballan** [comment]
Apparently the version of AD on @klauer's test stand *has* these attrs, whereas the one @arkilic, @dchabot and I were testing on does not.

Any opinions on which version of AD we should standardize the facility on? Any reason not to use "latest"? cc @stuwilkins

**Comment by klauer** [comment]
For reference, I am using 1-9-1 from the Debian packages and @dchabot's example IOC startup script.

**Comment by klauer** [comment]
Further reference, current (2-x) ADCore database templates:

`NDPluginBase` has the `Queue` stuff: https://github.com/areaDetector/ADCore/blob/master/ADApp/Db/NDPluginBase.template#L143

NDPluginBase includes `NDArrayBase`, which has the `Pool` stuff: https://github.com/areaDetector/ADCore/blob/master/ADApp/Db/NDArrayBase.template#L352

This leads me to believe (as I thought I had verified in the past) that the PVs for plugins have not changed... ?

Can you please look further into this @dchabot?

**Comment by tacaswell** [comment]
Closing as not helpful
